content/applications/hr/lunch/management.rst

```diff
@@ -0,0 +1,158 @@
+:guilabel:`Confirm Orders` button remains on the vendor line.
+
+.. image:: management/confirm.png
+   :align: center
+   :alt: The Today's Orders dashboard, with the two different ways to confirm an order highlighted.
+
+.. example::
+   A vendor receives an order for three pizzas and an order of garlic knots. When the delivery is
+   made to the company, the *Lunch* manager notices the garlic knots are missing.
+
+   The manager first marks the three pizzas as received by individually confirming the products with
+   the :guilabel:`✔️ Confirm` icon at the end of each product line.
+
+   Later, when the vendor delivers the garlic knots, the manager can either click the :guilabel:`✔️
+   Confirm` icon at the end of the line for the garlic knots, or click the :guilabel:`Confirm
+   Orders` button that appears next to the vendor's name and phone number.
+
+.. _lunch/notify:
+
+Notify employees
+----------------
+
+After products are received and the orders are confirmed, the employees must be informed that their
+orders have been delivered and are ready to be picked up.
+
+Unlike sending and confirming orders, notifications must be sent individually and cannot be sent in
+a batch.
+
+To notify the user their product has arrived, click the :guilabel:`✉️ Send Notification` icon at the
+end of each product line. An email is sent to the user informing them their products have been
+delivered.
+
+.. important::
+   It is important for each individual company to configure their day-to-day workflow to include
+   the necessary steps for the *Lunch* app. That includes designating people to be responsible for
+   sending and confirming orders, informing employees when orders have arrived, and determining
+   where orders are stored while waiting to be picked up by employees. These are parts of the
+   *Lunch* app flow that are not configured within Odoo and are determined by each individual
+   company.
+
+Control Vendors
+===============
+
+All orders for all vendors, both past and present, can be found in the *Control Vendors* dashboard.
+To access these records, navigate to :menuselection:`Lunch app --> Manager --> Control Vendors`.
+
+All orders appear in a list view, grouped alphabetically by :guilabel:`Vendor`. The list loads with
+all vendors expanded to show all order lines for every vendor by default.
+
+The list displays the :guilabel:`Order Date`, :guilabel:`Vendor`, :guilabel:`Product`,
+:guilabel:`Extras`, :guilabel:`Notes`, :guilabel:`User`, :guilabel:`Lunch Location`,
+:guilabel:`Price`, and :guilabel:`Status`. If in a multi-company database, a :guilabel:`Company`
+field appears.
+
+Orders can be :ref:`canceled <lunch/cancel>`, :ref:`sent to the vendor <lunch/send-orders>`,
+:ref:`confirmed <lunch/confirm-orders>` upon arrival, and :ref:`employees can be notified
+<lunch/notify>` using the same method as on the :ref:`Today's Orders <lunch/todays-orders>`
+dashboard.
```

Comment:
OOooh I really like this summary with the ref links to the sections. I'd actually move this up to the very beginning of your doc, in line , right after this line:
> ... their orders have arrived. This can be the same person.

Reply:
I did a mix- I added it to both, and updated the text it so it makes sense. I wanted to make sure that people understood it's best to use the Today's Orders dashboard (easier and more clear) than the Contorl Vendors- BUT that they can do the same thing.